### PR TITLE
Homework9: Marie Ambrazhei

### PR DESCRIPTION
**Pull request by MarieAmbrazhei**

### CODE REVIEW

File: homework9.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        else:
+            even.append(int(card_number_reverted[elem]))
+
+    final_sum = sum(odd + even)
+    if final_sum % 10 == 0:
+        return True
+    return False
+
+
+print(validate_card('4561261212345464'))
```
Comment (Stml89) [inline homework9.py:37]: 1/ use `assert` instead of `print()`
2/ card number - int, not str

File: homework9.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+# Вы можете удалить 3 из массива, чтобы получить строго возрастающую
+# последовательность [1, 2]. Альтернативно можно убрать 2, чтобы получить
+# строго возрастающую последовательность [1, 3].
+
+
+def can_be_ascending_by_removing_one(seq_int: list) -> bool:
+    """
+    Get a strictly ascending sequence by removing only one element.
+    """
+    if len(seq_int) - len(set(seq_int)) > 1:
```
Comment (Stml89) [inline homework9.py:62]: what is the porpoise of this `if` condition?

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: homework9.py (hunk 1 above)
Comment (MarieAmbrazhei) [reply homework9.py:37]: 1. added assert
2. fixed datatype

File: homework9.py (hunk 2 above)
Comment (MarieAmbrazhei) [reply homework9.py:62]: the purpose is to skip verification if our list contains more than 1 duplicate. For instance, we have [1, 2, 1, 2], there are two duplicates so if we delete one element, we still have duplicate in our list, thus  given list cant have strictly ascending order.

**Review by Stml89 — APPROVED** [review]
Fix comments, and you can merge PR

File: homework9.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+        if final_sum % 10 == 0:
+            return True
+        return False
+    print(f'Card number should have datatype int '
+          f'current type is {type(card_number)}')
+    return False
+
+
+# assert validate_card(4561261212345464), 'Please provide correct card number'
+# assert validate_card(4561261212345467)
```
[the remark this answers is not in the document]
Comment (MarieAmbrazhei) [reply homework9.py:41]: sorry, fixed it.

File: homework9.py [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+                return False
+
+            if i > 0 and seq_int[i - 1] >= seq_int[i + 1] and i + 2 < len(
+                    seq_int) and seq_int[i] >= seq_int[i + 2]:
+                return False
+
+    return True
+
+
+print(can_be_ascending_by_removing_one([1, 2, 3]))
```
[the remark this answers is not in the document]
Comment (MarieAmbrazhei) [reply homework9.py:83]: done.

File: homework9.py [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+def find_opposite_number(n: int, first_number: int) -> int:
+    """
+    Find the the radially opposite number.
+    """
+    degree_between_numbers = 360 / n
+    opposite_number = (180 / degree_between_numbers + first_number) % n
+    return int(opposite_number)
+
+
+print(find_opposite_number(10, 9))
```
[the remark this answers is not in the document]
Comment (MarieAmbrazhei) [reply homework9.py:108]: done.

File: homework9.py
[the remark this answers is not in the document]
Comment (MarieAmbrazhei) [reply homework9.py:1]: Got it! The next homework will be done using separate files.

[the remark this answers is not in the document]
Comment (MarieAmbrazhei) [reply homework9.py:1]: *This HW was separated as well

**Pull request merged**